st2tests/st2tests/fixtures/packs/dummy_pack_9/pack.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name : dummy_pack_9_deps
+description : dummy pack with pack and engine dependencies
+version : 0.1.0
+author : st2-dev
+email : info@stackstorm.com
+dependencies :
+  - core=0.2.0
+system :
+  stackstorm : 1.6dev
```

Comment:
> Looks like we’ll need a range and not a simple number for this. So when we actually start validating these fields, we should make sure that we don't lock ourselves on specifying an exact version.

The number was supposed to be a minimum version, but you're right, we could find ourselves deprecating or removing stuff, so I'm adding full range support.